Dockerfile

```diff
@@ -1,13 +1,5 @@
-### This is a generated file from Dockerfile.in ###
-#@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_8_golang_1.18)
-FROM registry.redhat.io/ubi8/go-toolset:1.18.9 AS builder
-
-ENV BUILD_VERSION=${CI_CONTAINER_VERSION}
-ENV OS_GIT_MAJOR=${CI_X_VERSION}
-ENV OS_GIT_MINOR=${CI_Y_VERSION}
-ENV OS_GIT_PATCH=${CI_Z_VERSION}
-ENV SOURCE_GIT_COMMIT=${CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_COMMIT}
-ENV SOURCE_GIT_URL=${CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_URL}
+FROM registry.redhat.io/ubi9/go-toolset:1.18.10-4 AS builder
```

Comment:
We're using Go 1.20.3 in midstream already. There does not seem to be a `go-toolset` image for that yet, so maybe `docker.io/library/golang` is an alternative?